src/views/index.js

```diff
@@ -56,6 +56,8 @@ exports.setLanguage = language => {
 const markdownUrl = "https://commonmark.org/help/";
 const doctypeString = "<!DOCTYPE html>";
 
+const THREAD_PREVIEW_LENGTH = 3;
```

Comment:
I didn't see a configuration file for constants like this so I added it inside the module